### PR TITLE
Remove wildcard permission from clusterrole

### PR DESCRIPTION
**Pull request by zlayne**
Signed-off-by: Zack Layne <zlayne@redhat.com>

Related issue: https://github.com/stolostron/backlog/issues/22961

### CODE REVIEW

File: stable/console-chart/templates/console-clusterrole.yaml [hunk 1]
```diff
@@ -195,7 +195,20 @@ rules:
...
+  - channels
+  - gitopsclusters
+  - helmreleases
+  - placementrules
+  - placementbindings
+  - policies
+  - policyautomations
+  - policysets
+  - subscriptions
+  - subscriptionreports
```
[the remark this answers is not in the document]
Comment (zlayne) [reply stable/console-chart/templates/console-clusterrole.yaml:211]: Might be better, this is how it is already done in the backplane-operator files as well. https://github.com/stolostron/backplane-operator/pull/205/files

[the remark this answers is not in the document]
Comment (KevinFCormier) [reply stable/console-chart/templates/console-clusterrole.yaml:211]: Yeah, I'd rather keep it in-sync in that case.

**Review by KevinFCormier — APPROVED** [review]

**Pull request merged**